### DISCUSSION TITLE
Display in Xamarin.Forms is different than in .NET MAUI because of FlexLayout or AbsoluteLayout

### DISCUSSION THREAD

**Discussion by anzeko123** [dotnet/maui#16336] [Upgrading from Xamarin]
I recently just started with a mobile app migration from Xamarin.Forms to .NET MAUI. I successfuly converted some stuff to .NET MAUI but now I am stuck where I can't get the same display I got in Xamarin.Forms.

In .NET MAUI my text doesn't display on the button, a down arrow is missing and logo is bigger. When I copied I almost didn't do anything. The only thing I changed are some class names in xaml.cs file and dependencies. 

I also tried using Border instead of GestureFrame but it didn't work. Tried also different layout and paddings/margins and didn't have any results. Tries removing some stuff so I could get to the core of the problem but I just couldn't find it.

The page has also 2 ContentViews FlexButton and LookupControlSignIn. I found out that the problem is probably in the FlexButton since when I changed something it was only effected when I changed FlexButton.

I should also mention that I am using Prism framework for .NET MAUI.

All code is here:
[https://github.com/anzeko123/siht]
I figured out that the FlexButton breaks this stuff but I can't figure out what is wrong and how to fix it. Maybe I am wrong and am looking in wrong files.

I am also new to C# so I have a difficult time understanding everything. Any help would be very appreciated.

This is a picture of the .NET MAUI app:

![NET MAUI](https://github.com/dotnet/maui/assets/120727345/6f22bdc5-56ab-4b69-b558-db6495dc63e5)

And this is Xamarin.Forms app and how it should look like:
![Xamarin Forms](https://github.com/dotnet/maui/assets/120727345/1790882a-274d-40de-b80e-1150a7264453)

**Comment by mikeysouthwell** [comment]
There is a known issue with the <Border> class.

If you have a border inside a Frame and some views it can cause issues.

Try this layout:

```XAML
<Border>
    <Frame>
      <Grid>
        <Entry/>
        <Image/>
      </Grid>
   </Frame>
</Border>
```

**Reply by bradencohen** [reply]
Out of curiosity, what benefit does the `Frame` provide here? Couldn't you just lose it altogether?

**Reply by mikeysouthwell** [reply]
You are correct. You could lose the frame and treat the border like a "frame". I was more trying to illustrate that if he was to put the frame or the grid outside the border then the stuff inside the border would vanish from the app.
